packages/node_modules/@ciscospark/widget-message-meet/src/store.js

```diff
@@ -13,7 +13,7 @@ const devtools = window.devToolsExtension || (() => (noop) => noop);
 const middlewares = [thunk];
 
 if (process.env.NODE_ENV !== `production`) {
-  const createLogger = require(`redux-logger`); // eslint-disable-line global-require
+  const {createLogger} = require(`redux-logger`); // eslint-disable-line global-require
```

Comment:
Exports `logger` as default, which come with default options, but since we are creating our own, we can't use the default.